docs/recipes/docker-client/Dockerfile

```diff
@@ -14,6 +17,9 @@ RUN apt-get update \
 RUN npm install -g @lhci/cli@0.8.2
 RUN npm install -g lighthouse
 
+# Install puppeteer
+RUN npm install -g puppeteer
```

Comment:
you'll need to add this after the next RUN block or it won't be available

`RUN cd /home/lhci/reports && npm link puppeteer`

Reply:
can test it out with a script in `scripts/foo.js` and running `docker run -v "$(pwd)/lhci-data:/home/lhci/reports/.lighthouseci" -v "$(pwd)/scripts:/home/lhci/reports/scripts" lhci-client lhci collect --puppeteerscript=scripts/foo.js --url=https://example.com`